examples/widgets/annotated_cursor.py

```diff
@@ -0,0 +1,345 @@
+    library/string.html#formatstrings>`_, optional, default: "{0:.4g};{1:.4g}"
+        The displayed text is created by calling *format()* on this string
+        with the two coordinates.
+
+    offset : 2D array-like, optional, default: [5, 5]
+        The offset in display (pixel) coordinates of the text position
+        relative to the cross hair.
+
+    dataaxis : {"x", "y"}, optional, default: "x"
+        If "x" is specified, the vertical cursor line sticks to the mouse
+        pointer. The horizontal cursor line sticks to the plotted line
+        at that x value. The text shows the data coordinates of the plotted
+        line at the pointed x value. If you specify "y", it works vice-versa.
+        But: For the "y" value, where the mouse points to, there might be
+        multiple matching x values, if the plotted function is not biunique.
+        Cursor and text coordinate will always refer to only one x value.
+        So if you use the parameter value "y", ensure that your function is
+        biunique.
+
+    Other Parameters
+    ----------------
+    textprops : `matplotlib.text` properties as dictionay
+        Specifies the appearance of the rendered text object.
+
+    **cursorargs : `matplotlib.widgets.Cursor` properties
+        Arguments passed to the internal `~matplotlib.widgets.Cursor` instance.
+        The `matplotlib.axes.Axes` argument is mandatory! The parameter
+        *useblit* can be set to *True* in order to achieve faster rendering.
+
+    """
+
+    def __init__(self, line, numberformat="{0:.4g};{1:.4g}", offset=[5, 5],
+                 dataaxis='x', textprops={}, **cursorargs):
+        # The line object, for which the coordinates are displayed
+        self.line = line
+        # The format string, on which .format() is called for creating the text
+        self.numberformat = numberformat
+        # Text positio offset
+        self.offset = np.array(offset)
+        # The axis in which the cursor position is looked up
+        self.dataaxis = dataaxis
+
+        # First call baseclass constructor.
+        # Draws cursor and remembers backround for blitting.
+        # Saves ax as class attribute.
+        super().__init__(**cursorargs)
+
+        # The numbers displayed next to cursor. Default value.
+        self.textdata = [0, 0]
+        # Default value for position of text.
+        self.setpos(self.line.get_xdata()[0], self.line.get_ydata()[0])
+        # Create invisible animated text
+        self.text = self.ax.text(
+            self.ax.get_xbound()[0],
+            self.ax.get_ybound()[0],
+            "0, 0",
+            animated=bool(self.useblit),
+            visible=False, **textprops)
+        # The position at which the cursor was last drawn
+        self.lastdrawnplotpoint = None
+
+    def onmove(self, event):
+        """Overwritten draw callback for cursor. \
+        Called when moving the mouse."""
+
+        # Leave method under the same conditions as in overwritten method
```

Comment:
```suggestion
        # Leave method under the same conditions as in overridden method.
```

Reply:
Fixed in multiple spots in 4aef9a8dbd0e5b4489255c85e2d0a8f7d62b4c25

Reply:
Linebreak added in 4d3dde10acc80afc865315b6c14416c47049aaa7